### DISCUSSION TITLE
Workflow

### DISCUSSION THREAD

**Discussion by albert849** [xyflow/xyflow#4717] [Show and tell]
I want to see the layout from the beginning to the end

**Comment by albert849** [comment]
Ppl so do I

**Comment by bcakmakoglu** [comment]
You haven't written anything meaningful in this post at all.
Before opening a discussion or issue, take the time to properly formulate a question or request, please.
Nobody can read your mind and nobody knows what you're trying to do if you just write 1 word or a single sentence, that's simply not how it works.

Try to answer some basic questions for yourself when formulating a post, for example:
* What are you trying to do? (give as much detail as possible/necessary)
* What have you tried?
	* What hasn't worked if you have made attempts to solve the issue?
* A minimal reproduction of the issue or a description of the outcome you expect to have.

You cannot expect anyone to help you if you don't even respect others time to write a proper post.
